src/com/aca/Armine Arzumanyan/HomeWork_1/src/Gadget.java

```diff
@@ -0,0 +1,57 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class Gadget implements Item {
+    public String name;
+    public int code;
+    public int price;
+    public int recDuration;
+    public boolean mp4;
+    public boolean mp3;
+    public boolean hd;
+    public boolean vga;
+    static List<Gadget> allItems = new ArrayList<>();
+    static List<Gadget> soledGadgets = new ArrayList<>();
+
+    public static void sailItem(Gadget item) {
+        soledGadgets.add(item);
+        allItems.remove(item);
+    }
+
+    public Gadget(int code) {
+        this.code = code;
+    }
+
+    @Override
+    public int changePrice(int newPrice) {
+        price = newPrice;
+        return price;
+    }
+    public static int sumPrice(List<Gadget> gadgetList) {
```

Comment:
let's imagine that you will have several managers and each of them should have own product list to manage and implement your structure keeping this notice in mind.

PS you should avoid these static instances